Doc/library/math.rst

```diff
@@ -393,13 +393,12 @@ Power and logarithmic functions
    .. versionadded:: 3.2
 
 
-.. function:: log(x[, base])
+.. function:: log(x, base=None)
 
-   With one argument, return the natural logarithm of *x* (to base *e*).
-
-   With two arguments, return the logarithm of *x* to the given *base*,
-   calculated as ``log(x)/log(base)``.
+   Return the logarithm of *x* to the given *base*.
 
+   If the *base* is not specified or is None, returns the natural
```

Comment:
Suggest dropping the `or is None`, for readability - it's clear from context that a value of `None` would be fall under the "not specified" rules.
```suggestion
   If the *base* is not specified, returns the natural
```